app/src/sharedTest/java/org/oppia/app/ongoingtopiclist/OngoingTopicListActivityTest.kt

```diff
@@ -132,14 +136,27 @@ class OngoingTopicListActivityTest {
     }
   }
 
+  @Test
+  fun testTopicPracticeFragment_loadFragment_changeOrientation_textIsCorrect() {
```

Comment:
This test case is failing.

Please take reference from this test case: `testOngoingTopicList_checkItem1_titleIsCorrect`

Also, in your previous PR which you have closed, I had mentioned some changes also, please do those changes here too.

Reply:
Does I have to do the same thing as in  `testOngoingTopicList_checkItem1_titleIsCorrect` . Like get to the position of recycler view and then check if it contains substring or not?
@rt4914

Reply:
Yes but with one step added for orientation change.
`testOngoingTopicList_changeConfiguration_checkItem1_titleIsCorrect`

Reply:
Okayy

Reply:
@rt4914 
Is this okay?
`  @Test
  fun testTopicPracticeFragment_loadFragment_changeOrientation_textIsCorrect() {
    launch<OngoingTopicListActivity>(createOngoingTopicListActivityIntent(internalProfileId)).use {
      onView(withId(R.id.ongoing_topic_list)).perform(
        scrollToPosition<RecyclerView.ViewHolder>(
          1
        )
      )
      onView(atPositionOnView(R.id.ongoing_topic_list, 1, R.id.topic_name_text_view)).check(
        matches(
          withText(containsString("Ratios"))
        )
      )
    }
  }`

Reply:
@virajpatel2411 Actually it is not correct to check like this, make a commit and then I will check this in android studio.

Reply:
Okayy

Reply:
I have made a commit. Please check @rt4914.